### DISCUSSION TITLE
Ignore files that cannot be accessed

### DISCUSSION THREAD

**Discussion by zurek-piotr** [opcodesio/log-viewer#262] [Ideas]
It would be great if there was an option like `hide_unknown_files` to ignore files that cannot be accessed.

<img width="650" alt="image" src="https://github.com/opcodesio/log-viewer/assets/49003552/71b37a50-a665-41a8-8ffe-362bc1ac72d6">

Thanks for awesome work! ❤️

**Comment by arukompas** [comment]
hey @zurek-piotr , could you please try the latest release, **v3.0.3** please? Let me know if that helps :)

**Reply by zurek-piotr** [reply]
Now works great 👍
